client/src/main/scala/io/delta/sharing/client/DeltaSharingFileSystem.scala

```diff
@@ -65,6 +65,7 @@ private[sharing] class DeltaSharingFileSystem extends FileSystem {
 
   override def getUri(): URI = URI.create(s"$SCHEME:///")
 
+  // delta-sharing:///
```

Comment:
This comment is not clear. Consider expanding on it.

Reply:
updated.